src/formatDate.js

```diff
@@ -51,6 +51,50 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const oldSeparator = fromFormat[fromFormat.length - 1];
+  const newSeparator = toFormat[toFormat.length - 1];
+  const dateArray = date.split(oldSeparator);
+  const dateFormattedArray = [];
+
+  const dateObj = {};
+
+  dateArray.forEach((item, i) => {
+    dateObj[fromFormat[i][0]] = item;
```

Comment:
```suggestion
    dateObj[fromFormat[i]] = item;
```
it's pretty hard to read/understand if you do it [i][0], you can just use [i], no need to get only first digit

Reply:
if i get rid of [0] it will break all down

it's not for digits, but for letters. I've got object keys 'm', 'd', 'y' to not be concerned about which format of year (YY or YYYY) we've got. So,  i need only a first letter of the format for this solution

i can change that, but i'll need to implement new logic for the cycle, as i see..

Reply:
I see, then you can leave it just as it is